identity-credential/src/README.md

```diff
@@ -0,0 +1,161 @@
+IOTA Identity - Credentials 
+=== 
```

Comment:
Can we move this to `identity-credential/README.md`? It would be nice for those browsing on GitHub to see it too.

Reply:
Sure